### PR TITLE
docs: Describe messages descriptions better 

### PR DESCRIPTION
**Pull request by Carlos-Muniz**
**TL;DR -**

These messages have been updated to be more descriptive.
The message descriptions need to be more descriptive in order to
be easily translated (i18n) into other languages. 

**What changed?**
- *.messages.js descriptions
- Snapshots for tests about these messages

**Developer Checklist**
- [x] Test suites passing
- [x] Documentation and test plan updated, if applicable
- [x] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)

**Testing Instructions**

Review each of the descriptions to confirm that they add context
and are easier to understand by translators.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @Carlos-Muniz! I've created [OSPR-6248](https://openedx.atlassian.net/browse/OSPR-6248) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Review by sarina — COMMENTED** [review]
@Carlos-Muniz all your revised descriptions look great. The only thing needed is to address the linter failings.

File: src/data/constants/filters.messages.js [hunk 1]
```diff
@@ -4,37 +4,38 @@ const messages = defineMessages({
   assignment: {
     id: 'gradebook.GradesTab.FilterBadges.assignment',
     defaultMessage: 'Assignment',
-    description: 'Assignment FilterBadge label',
+    description: 'A label describing the notification under the \"Edit Filters\" button that shows by which of the course\'s assignments the view is being filtered.',
```
[the remark this answers is not in the document]
Comment (Carlos-Muniz) [reply src/data/constants/filters.messages.js]: I've updated these messages to pass the linter

**Review by sarina — APPROVED** [review]
@Carlos-Muniz - the bot wasn't reading your name properly - the CLA check should pass now when re-run.

You should squash your commits before merging. This PR probably only needs one commit.

This is a great runthrough of PRs: https://handbook.opencraft.com/en/latest/processes/pull_request/

"The best pull requests contain only a single, logical change – which means only a single, logical commit." - from https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/process/contributor.html

If you squash your commits and push up, you'll trigger a rebuild that should pass - and then you can go ahead & merge!

**Comment by openedx-webhooks** [comment]
Although this pull request is already merged, I'm still watching it for updates.

There is nothing you have to do. No action is needed from your side.  Thanks again for your contribution.

**Comment by openedx-webhooks** [comment]
Although this pull request is already merged, I'm still watching it for updates.

There is nothing you have to do. No action is needed from your side.  Thanks again for your contribution.
